### ISSUE TITLE
Write conversion for document link aria-label

### ISSUE THREAD

**Issue by greensomes** [DOI-ONRR/onrr.gov-site#3336]
Linked to #3335. Existing links should have aria-label updated from "Download <file type>" to "Download <link text>"

**Comment by AlexandracmONRR** [comment]
Related to our discussion for the research issue #3323

**Comment by greensomes** [comment]
Hey, @AlexandracmONRR! I wrote a script and ran it against the preview CMS. See attached screenshot. Please poke around to validate that everything looks good before I run it in dev and prod. Thanks!
<img width="992" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBL0ViQnc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--13f432037d08e2b704841503a0d120340db5bb93/Screenshot%202024-10-02%20at%208.33.22%E2%80%AFPM.png" alt="Screenshot 2024-10-02 at 8.33.22 PM.png" />

**Comment by AlexandracmONRR** [comment]
@greensomes I checked different links and they are reading correctly and without flags from ANDI. Thank you!!

**Comment by AlexandracmONRR** [comment]
Creating new PDF links in in-line link editor create aria labels correctly! Thanks!
<img width="1248" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNFVlQnc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--5802fc38086a823a3e1027e75b7621cb536ec161/image.png" alt="image.png" />

**Comment by AlexandracmONRR** [comment]
<img width="1252" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeGNnQnc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--68a2c11e1df490cc0b4bcb2e522e6235e709918b/image.png" alt="image.png" />
